frontend/src/api/program.tsx

```diff
@@ -1,15 +1,22 @@
 import axios from 'axios'
+import { ProgramType } from '../data/program'
 
-const getProgram = async () => {
+const getPrograms = async (): Promise<ProgramType[]> => {
   const url = `/api/programs`
-  const { data: response } = await axios.get(url)
-
-  return response.data
-    .map((p) => ({ id: p.id, ...p.attributes }))
-    .filter((p) => {
-      if (!p.expirationDate) return true
-      return new Date(p.expirationDate) > new Date()
+  return axios
+    .get(url)
+    .then((response) => {
+      return response.data.data
+        .map((p) => ({ id: p.id, ...p.attributes }))
+        .filter((p) => {
+          if (!p.expirationDate) return true
```

Comment:
Can this expiration date filter be moved at the Strapi api level? It would be more efficient to not transport such programs so that they're filtered in the frontend.

Reply:
done